src/olympia/addons/tasks.py

```diff
@@ -23,9 +23,6 @@
 from olympia.lib.es.utils import index_objects
 from olympia.versions.models import Version
 
-# pulling tasks from cron
-from . import cron  # noqa
```

Comment:
I've been tracing the introduction of this import all the way from d0aa9d2915cf78761ebf53e4047e37d6f409d6b8, 7 years ago, and it doesn't make any sense... glad to see it gone :)